Backend/Tournament_User/TournamentUserMVSC/server.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+const tournamentUserRoutes = require('./routes/tournamentUserRoutes');
+const errorHandler = require('./middlewares/errorHandler');  // Import error-handling middleware
+const successHandler = require('./middlewares/successHandler');// Import success-handling middleware
+
+
+// Middleware to parse incoming JSON
+app.use(express.json());
+
+// Tournament user routes
+app.use('/api', tournamentUserRoutes);
+
+// Use the error-handling middleware after routes
+app.use(errorHandler);
+
+// use the success-handling middleware
+app.use(successHandler);
```

Comment:
just to confirm, your successHandler is placed after your errorHandler, but `next()` is called in the successHandler?

Reply:
swap order alr boss :(